app/static/joke.js

```diff
@@ -0,0 +1,14 @@
+function makejoke() {
+  fetch('/joke')
+    .then(response => response.json())
+    .then(data => alert(data.text));
+}
+
+
+function addEventsAfterPageLoaded() {
+  const element = document.getElementById("ajoke");
+  element.addEventListener("click", makejoke, false);
+  };
```

Comment:
This shouldn't be indented.
Also, please remove the `;`

Reply:
Done